crates/block-producer/src/mempool/state.rs

```diff
@@ -187,10 +187,18 @@ impl AccountUpdates {
         self.to.insert(to, id);
     }
 
+    /// Returns the node ID that updated this account's commitment to the given value.
+    ///
+    /// In other words, this returns the node ID where `node.to == from`. This infers the
+    /// parent-child relationship where `parent.to == child.from`.
     fn parent(&self, from: &Word) -> Option<&NodeId> {
         self.to.get(from)
     }
 
+    /// Returns the node ID that consumed the given commitment,
+    ///
+    /// In other words, this returns the node ID where `node.from == to`. This infers the
+    /// parent-child relationship where `parent.to == child.from`.
```

Comment:
```suggestion
    /// Returns the node ID that consumed the given commitment.
    ///
    /// In other words, this returns the ID of the `node` where `node.from == to`. This infers the
    /// parent-child relationship where `parent.to == child.from`.
```

Nit: Maybe like this? Otherwise, sounds great!